src/main/webapp/assets/js/function.js

```diff
@@ -0,0 +1,20 @@
+function faq() {
```

Comment:
Use a verb for the function name.

Function names should be verbs,
Variables should be nouns.